pages/home/home.html

```diff
@@ -0,0 +1,82 @@
+        <h5>The Oppia Foundation is working to change that.</h5>
+    </md-content>
+
+    <md-content>
+      <h3>A new approach to online learning </h3>
+      <p>
+        Oppia is a free and open-source online learning platform for collaboratively creating and sharing interactive lessons.
+      </p>
+      <p>
+        Oppia’s lesson builder allows educators to provide students with targeted, personalized   feedback that simulates conversation with an intelligent, virtual tutor.
+      </p>
+    </md-content>
+
+    <md-content>
+      <h3>Impacting students around the world</h3>
+      <p>
+        Oppia was built to meet the unique needs of the developing world, where educational resources are scarce and methodologies are outdated. 
+      </p>
+      <p>
+        Our storytelling-based approach to learning requires less bandwidth than educational videos, making it accessible in areas with limited internet connections. Adaptive lessons with personalized feedback create a safe, interactive learning environment otherwise unavailable to underserved communities.
+      </p> 
+      <p>
+        By uniting a global community of volunteers and educators, Oppia’s lessons are continually translated into new languages and adapted for different cultures.
+      </p>
+    </md-content>
+
+    <md-content>
+      <h5>
+        Since launching Oppia.org in 2015, over 800,000 learners in more than 20 countries have used our platform to explore, grow, and learn.
+      </h5>
+    </md-content>
+      
+    <md-content>
+      <h3>Get involved</h3>
+      <p>
+        The Oppia Foundation is a community of open-source developers, researchers, and educators around the world who volunteer to make our vision a reality.
+      </p>
+      <p>
+        We are looking for organizations that want to join us on our journey to transform how students learn online.
+      </p>
+      <p>
+        With your help, we aim to improve our platform by conducting randomly controlled trials, fund a dedicated team of lesson creators, and manage the costs of scaling the Oppia project to serve learners across the world.
+      </p>
+    </md-content>
+
+    <md-content>
+      <h3>Partner with us</h3>
+      <p>
+        We are looking for organizations that want to join us on our journey to transform how students learn online.
+      </p>
+      <p>
+        With your help, we aim to improve our platform by conducting randomly controlled trials, fund a dedicated team of lesson creators, and manage the costs of scaling the Oppia project to serve learners across the world.
+      </p>
+    </md-content>
+
+    <md-content>
+      <h3>Make a donation</h3>
+      <p>
+        By making a donation to The Oppia Foundation, you are helping to change millions of students’ lives. Together, we can revolutionize the way students around the world use the internet to learn.
+      </p>
+    </md-content>
+  </md-content>
+</section>
+
+<style>
+
+  .welcome_image {
```

Comment:
.home-page-welcome-image

(typically, use kebab-case for CSS, and let's prefix it with the page that the CSS is on unless you want to share the CSS with other pages.)

Reply:
Fixed :)